test/distributed/elastic/rendezvous/dynamic_rendezvous_test.py

```diff
@@ -1795,14 +1804,27 @@ def test_use_agent_store_is_disabled(self):
 
     @patch.object(dist, "PrefixStore")
     def test_share_tcp_store_from_backend(self, prefix_store_class_mock):
-        prefix_store = Mock(spec=dist.PrefixStore)
-        prefix_store_class_mock.return_value = prefix_store
+        expected_addr = "expected_address"
+        expected_port = 54231
```

Comment:
Not related to this PR, maybe define some constants so that later comers do not need to search for each instance of 54321 if they want to modify it?

Reply:
I agree that we will need to make it constant and I will do that. But this change does not fall into that you said, even with the constant, we are not changing it everywhere. This is an expected behavior change.